### PR TITLE
{math}[foss/2022a] JAGS v4.3.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
himem04.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/5b9edef5f499aef2c245d67c6a6623aa for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a990c1082e4a3ab7adf3e6ae639ac156 for a full test report.

File: easybuild/easyconfigs/j/JAGS/JAGS-4.3.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sources = [SOURCE_TAR_GZ]
+checksums = ['f9258355b5e9eb13bd33c5fa720f0cbebacea7d0a4a42b71b0fb14501ee14229']
+
+configopts = ' --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'libexec/%(namelower)s-terminal', 'lib/libjags.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
```
Comment (verdurin) [inline easybuild/easyconfigs/j/JAGS/JAGS-4.3.1-foss-2022a.eb:29]: @sassy-crick don't suppose you feel like adding `sanity_check_commands`?

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ninhursaga.uio.no - Linux RHEL 9.1 (Plow), x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 525.60.13, Python 3.9.14
See https://gist.github.com/5ea28fedf1804cfe13f5bafc3291cd02 for a full test report.

File: easybuild/easyconfigs/j/JAGS/JAGS-4.3.1-foss-2022a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/j/JAGS/JAGS-4.3.1-foss-2022a.eb:29]: sanity check added in 6ff63ca6f3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3161.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/c3a3a28914bff9a99e58f9119ec5450c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**